source/configuration_manual/namespace/index.rst

```diff
@@ -87,36 +104,52 @@ When ``yes`` namespace will be hidden from IMAP ``NAMESPACE`` command.
 -------------------------------
 
  - Default: ``no``
- - Values: :ref:`boolean`
+ - Values:  :ref:`boolean`
+
+If :ref:`setting-namespace_location` fails to load, by default the entire
+session will fail to start.
 
-If namespace :ref:`setting-namespace_location` fails to load, this namespace will not be included.
-Otherwise the whole sessio will fail to start.
+If this is set, this namespace will be ignored instead.
 
 
 .. _setting-namespace_inbox:
 
 ``namespace/inbox``
 -------------------
 
- - Default: ``no``
- - Values: :ref:`boolean`
+- Default: ``no``
+- Values:  :ref:`boolean`
 
-When ``yes`` this namespace will be considered the one holding the ``INBOX`` folder.
-There can be only one namespace like this.
+If set, this namespace will be considered the one holding the INBOX folder.
+
+There can be only one namespace defined like this.
 
 
 .. _setting-namespace_list:
 
 ``namespace/list``
 ------------------
 
- - Default: `yes`
- - Values: :ref:`boolean`
+- Default: ``yes``
+- Values:  ``yes``, ``no``, or ``children``
+
+Include this namespace in LIST output when listing its parent's folders.
+
+Options:
+
+============= ==============================================================
+Value         Description
+============= ==============================================================
+``children``  Namespace prefix list listed only if it has child mailboxes.
+``no``        Namespace and mailboxes not listed.
```

Comment:
I guess the description above and below explains this further, but since nobody ever reads anything but the bare minimum, I think this could still be clarified " ... not listed, unless listing requests explicitly mailboxes under the namespace prefix."